tools/bumper/cnao_repo_commands.go

```diff
@@ -212,6 +215,134 @@ func (cnaoRepoOps *gitCnaoRepo) collectModifiedToTreeList(allowedList []string)
 	return entries, nil
 }
 
+// getBumpBranch creates a new bump branch.
+// Since we don't want to use already open branch
+// that may hold outdated/needs rebasing changes
+// we concat a random string to the end of the branch name
+func (cnaoRepoOps *gitCnaoRepo) getBumpBranch(prTitle string) (*github.Reference, string, error) {
+	branchBaseName := strings.Replace(strings.ToLower(prTitle), " ", "_", -1)
+	branchOpenMaxTries := 10
+	for branchIdx := 0; branchIdx < branchOpenMaxTries; branchIdx++ {
+		branchNameWithRandSuffix := fmt.Sprintf("%s_%s", branchBaseName, rand.String(5))
+		branchRef, _, err := cnaoRepoOps.githubInterface.GetRef(cnaoRepoOps.getCnaoRepoOwnerFromUrl(), cnaoRepoOps.getCnaoRepoNameFromUrl(), "refs/heads/"+branchNameWithRandSuffix)
```

Comment:
since githubInterface.GetRef already an interface funtion (and not a direct all to githubApi), I can just change it's name to `getBrnachRef` like your suggest. is that ok?

Reply:
also changed `githubInterface` function names to be private in a separate commit

Reply:
yep, sounds good. maybe there is one more thing we can be done here. see https://github.com/kubevirt/cluster-network-addons-operator/pull/509#discussion_r507686112